arm-datalake-analytics/job/2016-11-01/examples/AdlRecurrenceGet.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parameters": {
+    "api-version": "2016-11-01",
+    "accountName": "contosoadla",
+    "recurrenceId": "8b3e4d71-022d-4cf4-91af-703b954d7b95"
```

Comment:
this parameter is called recurrenceIdentity in the spec https://github.com/Azure/azure-rest-api-specs/pull/1272/files#diff-c3357f2ab384c3b8ab52d353d8b2724eR187 (error https://travis-ci.org/Azure/azure-rest-api-specs/jobs/238436043#L1120)